### ISSUE TITLE
A list of current map issues

### ISSUE THREAD

**Issue by CoWinkKeyDinkInc** [CoWinkKeyDinkInc/CardinalPGM-Maps#3]
- [x] Unsure if the Jungle Beat used is the Tourney version, or the new version that reduces rushing or something
- [x] Ring Race wool rooms regions are off, and the chests located in them are facing the wrong way.
- [x] Swamped has no spawn-kit at all, regions are off, and the core cannot be leaked as the lava does not flow.  Also, all chests in Swamped contain nothing.
- [x] Swamped Mini also has no spawn-kit and lava does not flow.
- [x] In Teneborus, chests need to be rotated, and the portals do not work, also some trapdoors and stone stairs in front lines are immune to TNT.  Could be another region problem.

**Comment by Prodigy4532** [comment]
Ring race fixed no?

**Comment by Prodigy4532** [comment]
i go to work on the map `Teneborus`

**Comment by CoWinkKeyDinkInc** [comment]
The chests are still facing the wrong way, I think.

**Comment by Lmmb74** [comment]
It seems that everything here is done, so you should close this issue.

**Comment by Lmmb74** [comment]
This issue should be closed.

**Comment by CoWinkKeyDinkInc** [comment]
I didn't look over them yet, have you?

**Comment by Lmmb74** [comment]
Yup. Also checked Prodigy commits. The errors you mentioned are solved. But I did not check Tenebrous chests, would you mind doing it?

**Comment by CoWinkKeyDinkInc** [comment]
I'll do the chests.
